src/WinUIExSample/MainWindow.xaml.cs

```diff
@@ -170,5 +170,30 @@ private void WindowMessageReceived(object sender, WindowMessageEventArgs e)
         {
             Log(e.Message.ToString());
         }
+
+        private void MoveToPoint_Click(object sender, RoutedEventArgs e)
+        {
+            this.Move(x_box.Value, y_box.Value);
+        }
+
+        private void CentreOnPoint_Click(object sender, RoutedEventArgs e)
+        {
+            this.CenterOnPoint(x_box.Value, y_box.Value);
+        }
+
+        //private void moveWindowRadioButtons_Checked(object sender, RoutedEventArgs e)
+        //{
+        //    var rb = sender as RadioButton;
+        //    HwndExtensions.Positions position = (HwndExtensions.Positions)Enum.Parse(typeof(HwndExtensions.Positions), rb.Content.ToString());          
+        //    this.MoveToSelectedPosition(position);
+
+        //}
+
```

Comment:
Dead code

Reply:
Understood, I didn't know if you'd prefer radio buttons to buttons, but I'll delete it.